code/game/machinery/adv_med.dm

```diff
@@ -380,156 +380,146 @@
-					infected = "Mild Infection+:"
+					ailments |= "Mild Infection+"
 				if(INFECTION_LEVEL_ONE + 300 to INFECTION_LEVEL_ONE + 400)
-					infected = "Mild Infection++:"
+					ailments |= "Mild Infection++"
 				if(INFECTION_LEVEL_TWO to INFECTION_LEVEL_TWO + 200)
-					infected = "Acute Infection:"
+					ailments |= "Acute Infection"
 				if(INFECTION_LEVEL_TWO + 200 to INFECTION_LEVEL_TWO + 300)
-					infected = "Acute Infection+:"
+					ailments |= "Acute Infection+"
 				if(INFECTION_LEVEL_TWO + 300 to INFECTION_LEVEL_TWO + 399)
-					infected = "Acute Infection++:"
+					ailments |= "Acute Infection++"
 				if(INFECTION_LEVEL_TWO + 400 to INFINITY)
-					infected = "Septic:"
+					ailments |= "Septic"
 
 			var/unknown_body = 0
 			for(var/I in e.embedded_objects)
 				unknown_body++
 
 			if(unknown_body || e.hidden)
-				imp += "Unknown body present:"
-			if(!AN && !open && !infected && !imp)
-				AN = "None:"
-			dat += "<td>[e.name]</td><td>[e.burn_dam]</td><td>[e.brute_dam]</td><td>[robot][bled][AN][splint][burn_wound][ointment][open][infected][imp][internal_bleeding][lung_ruptured][dead]</td>"
+				ailments |= "Unknown body present"
+
+			dat += "<td>[e.name]</td>"
+			dat += "<td>[e.burn_dam]</td>"
+			dat += "<td>[e.brute_dam]</td>"
+			dat += "<td>[jointext(ailments, "<br>")]</td>"
 			dat += "</tr>"
+		dat += "</table>"
+		dat += "<br>"
+
+		dat += "<table border='1' style='width:100%'>"
+		dat += "<tr>"
+		dat += "<th style='width:30%'>Organ</th>"
+		dat += "<th style='width:10%'>Damage</th>"
+		dat += "<th>Injuries</th>"
+		dat += "</tr>"
+
 		for(var/obj/item/organ/internal/i in occupant.internal_organs)
-			var/mech = i.desc
-			var/infection = "None"
+			var/infection = ""
 			var/dead = ""
 			if(i.status & ORGAN_DEAD)
-				dead = "DEAD:"
+				dead = "Dead"
 			switch(i.germ_level)
 				if(1 to INFECTION_LEVEL_ONE + 200)
-					infection = "Mild Infection:"
+					infection = "Mild Infection"
 				if(INFECTION_LEVEL_ONE + 200 to INFECTION_LEVEL_ONE + 300)
-					infection = "Mild Infection+:"
+					infection = "Mild Infection+"
 				if(INFECTION_LEVEL_ONE + 300 to INFECTION_LEVEL_ONE + 400)
-					infection = "Mild Infection++:"
+					infection = "Mild Infection++"
 				if(INFECTION_LEVEL_TWO to INFECTION_LEVEL_TWO + 200)
-					infection = "Acute Infection:"
+					infection = "Acute Infection"
 				if(INFECTION_LEVEL_TWO + 200 to INFECTION_LEVEL_TWO + 300)
-					infection = "Acute Infection+:"
+					infection = "Acute Infection+"
 				if(INFECTION_LEVEL_TWO + 300 to INFECTION_LEVEL_TWO + 399)
-					infection = "Acute Infection++:"
+					infection = "Acute Infection++"
 				if(INFECTION_LEVEL_TWO + 400 to INFINITY)
-					infection = "Septic:"
-
+					infection = "Septic"
 			dat += "<tr>"
-			dat += "<td>[i.name]</td><td>N/A</td><td>[i.damage]</td><td>[infection]:[mech][dead]</td><td></td>"
+			dat += "<td>[i.name]</td>"
+			dat += "<td>[i.damage]</td>"
+			dat += "<td>[infection]<br>[dead]</td>"
```

Comment:
The `<br>` here is unconditional, so you get the ghost line breaks visible in the bottom-right part of your 2nd screencap; might wanna turn this line into a similar `ailments` list that's then `jointext`'d as with the bodyparts table

Reply:
Updated this with commit [bf0e438](https://github.com/ParadiseSS13/Paradise/pull/21280/commits/bf0e438cdd13ae2016a64d2fab8af14b1be54f94) 😄